blockchain/fullblocktests/params.go

```diff
@@ -365,6 +365,33 @@ var regNetParams = &chaincfg.Params{
 			},
 			StartTime:  0,             // Always available for vote
 			ExpireTime: math.MaxInt64, // Never expires
+		}, {
+			Vote: chaincfg.Vote{
+				Id:          chaincfg.VoteIDAutoRevocations,
+				Description: "Enable automatic ticket revocations as defined in DCP0009",
+				Mask:        0x0006, // Bits 1 and 2
```

Comment:
`0x0600, // Bits 5 and 6`

Reply:
Updated, though I believe it should be `0x0060`.